views/components/embedsummary.js

```diff
@@ -0,0 +1,18 @@
+import React from "react-native";
+import AppText from "./app-text";
+
+const {
+	View
+} = React;
+
+export default class EmbedSummary extends React.Component {
+	render(){
+		return(
+			<View>
+				{this.props.embed.description ?
+				(<AppText numberOfLines = {2}><AppText style={{fontWeight:'bold'}}>Description : </AppText>{this.props.embed.description}</AppText>):null
```

Comment:
Remove additonal <AppText> component

Move styles out of the component into a `StyleSheet.create` call. React does some optimizations with this.

Remove `Description: `